README.md

```diff
@@ -51,13 +56,64 @@ To ensure this, compare the output of `docker --version` and `docker-compose --v
 ## Configuration
 
 You can configure the app using environment variables. These are defined in `netbox.env`.
+Read [Environment Variables in Compose][compose-env] to understand about the various possibilities to overwrite these variables.
+(The easiest solution being simply adjusting that file.)
+
+To find all possible variables, have a look at the [configuration.docker.py][docker-config] and [docker-entrypoint.sh][entrypoint] files.
+Generally, the environment variables are called the same as their respective Netbox configuration variables.
+Variables which are arrays are usually composed by putting all the values into the same environment variables with the values separated by a whitespace ("` `").
+For example defining `ALLOWED_HOSTS=localhost ::1 127.0.0.1` would allows access to Netbox through `http://localhost`, `http://[::1]` and `http://127.0.0.1`.
+
+[compose-env]: https://docs.docker.com/compose/environment-variables/
+
+### Production
+
+The default settings are optimized for (local) development environments.
+You should therefore adjust the configuration for production setups, at least the following variables:
+
+* `ALLOWED_HOSTS`: Add all URLs that lead to your netbox instance.
+* `DB_*`: Use a persistent database.
+* `EMAIL_*`: Use your own mailserver.
+* `MAX_PAGE_SIZE`: Use the recommended default of 1000.
+* `SUBERUSER_*`: Only define those variables during the initial setup, and drop them once the DB is set up.
```

Comment:
is SUBERUSER deliberate? or should it be superuser?

Reply:
It's deliberate, as those are environment variables and therefore capital by convention.

Reply:
not capitalization, spelling, suBeruser